tests/ssl-opt.sh

```diff
@@ -1832,6 +1839,7 @@ run_test    "Connection ID: Cli+Srv enabled, Cli+Srv CID empty, AES-128-CCM-8" \
             -C "Use of Connection ID has been negotiated"
 
 requires_config_enabled MBEDTLS_SSL_DTLS_CONNECTION_ID
+requires_ciphersuite_enabled TLS-ECDHE-ECDSA-WITH-AES-128-CBC-SHA256
```

Comment:
Note: this is also something we could be picking up from the command line, for example by expanding `check_cmdline_compat()`. However, the current approach is fine as well, so I'm not requesting any change, just thinking out loud :)